questions.md

```diff
@@ -0,0 +1 @@
+Do I have to use the command line? Usually I use GitHub Desktop and that has been great (and much easier to use).
```

Comment:
You can use Git with whatever desktop client you'd like! GitHub Desktop is a great option! The training uses the command line because it's the most comprehensive way to learn _exactly_ what is happening under the hood. Once you have those basics down, a GUI program like GitHub Desktop can often save time.